### PR TITLE
Add kmertools v0.1.0

### PR DESCRIPTION
**Pull request by Vini2**
Describe your pull request here

This PR adds [`kmertools`](https://github.com/anuradhawick/kmertools) to bioconda. `kmertools` is a k-mer based feature extraction tool designed to support metagenomics and other bioinformatics analytics.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by Vini2** [comment]
@BiocondaBot please fetch artifacts

**Comment by Vini2** [comment]
## Testing

```
conda install -c ./packages kmertools

which kmertools
/opt/miniconda3/bin/kmertools

kmertools --help

kmertools: DNA vectorisation

k-mer based vectorisation for DNA sequences for
metagenomics and AI/ML applications

Usage: kmertools <COMMAND>

Commands:
  comp  Generate sequence composition based features
  cov   Generates coverage histogram based on the reads
  min   Bin reads using minimisers
  ctr   Count k-mers
  help  Print this message or the help of the given subcommand(s)

Options:
  -h, --help
          Print help (see a summary with '-h')

  -V, --version
          Print version
```

Everything looks good!

**Comment by Vini2** [comment]
@BiocondaBot please add label

File: recipes/kmertools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  run_exports:
+    - {{ pin_subpackage('kmertools', max_pin="x") }}
+
+source:
+  url: https://github.com/anuradhawick/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - rust >=1.79
```
Comment (martin-g) [inline recipes/kmertools/meta.yaml:21]:
```suggestion
    - {{ compiler("rust") }}
```

File: recipes/kmertools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+{% set name = "kmertools" %}
+{% set sha256 = "3c8cf78c95ab0baf0f69e7bc631ea52a54733bed0122b9d20e76d49df5639a4c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
```
Comment (martin-g) [inline recipes/kmertools/meta.yaml:11]:
```suggestion
```
It builds platform specific binaries

File: recipes/kmertools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  build:
+    - rust >=1.79
+
+test:
+  commands:
+    - {{ name }} --help
+
+about:
+  home: https://github.com/anuradhawick/kmertools
+  license: GPL-3.0
```
Comment (martin-g) [inline recipes/kmertools/meta.yaml:29]:
```suggestion
  license: GPL-3.0-only
```
SPDX is

File: recipes/kmertools/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  license_family: GPL
```
Comment (martin-g) [inline recipes/kmertools/meta.yaml:30]:
```suggestion
  license_family: GPL3
```

**Review by martin-g — CHANGES_REQUESTED** [review]

**Comment by Vini2** [comment]
Hi @martin-g,

Thanks for reviewing and suggesting changes. I have made the requested changes. Can you please review them?

File: recipes/kmertools/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+{% set sha256 = "3c8cf78c95ab0baf0f69e7bc631ea52a54733bed0122b9d20e76d49df5639a4c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('kmertools', max_pin="x") }}
```
Comment (martin-g) [inline recipes/kmertools/meta.yaml:12]:
```suggestion
    - {{ pin_subpackage('kmertools', max_pin="x.x") }}
```

**Review by martin-g — APPROVED** [review]

**Pull request merged**